addons/website_livechat/static/src/web/persona_model_patch.js

```diff
@@ -0,0 +1,21 @@
+import { Persona } from "@mail/core/common/persona_model";
+
+import { patch } from "@web/core/utils/patch";
+import { deserializeDateTime } from "@web/core/l10n/dates";
+
+const { DateTime } = luxon;
+
+patch(Persona.prototype, {
+    get historyString() {
```

Comment:
```suggestion
    get historyLocalized() {
```
:pick: Since both are strings, I feel like the naming is confusing. What about this?

Reply:
Better, thanks for the suggestion ;)

Reply:
Of course, in master, we can keep "history" :)